config/default_config.json

```diff
@@ -3,8 +3,10 @@
     "0BSD",
     "Apache-1.0",
     "Apache-2.0",
+    "Artistic-2.0",
     "BSD-2-Clause",
     "BSD-3-Clause",
+    "BSD*",
```

Comment:
`BSD*` is just the license checker saying it's some sort of BSD, but doesn't know which one. It's not an actual SPDX identifier.

```suggestion
```


We should instead add an entry to https://github.com/Brightspace/d2l-license-checker/blob/b69d175d80e386b92fafcec5b45bb1469ca4cdb5/config/default_config.json#L27 below to tell license checker exactly what the library is (once someone knows on that thread).